dali/operators/util/get_property.cc

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/util/get_property.h"
+#include "dali/pipeline/data/types.h"
+
+namespace dali {
+
+DALI_SCHEMA(GetProperty)
+    .DocStr(
+        R"code(Returns a property of the tensor passed as an input.
+The type of the output will depend on the ``key`` of the requested property.
+
+Some of the properties are returned as a byte-array. The easiest way to convert them to string
+would be to )code")
+    .NumInput(1)
+    .NumOutput(1)
+    .AddArg("key",
+            R"code(Specifies, which property is requested. The following properties are supported:
```

Comment:
```suggestion
            R"code(Specifies, which property is requested.
            
The following properties are supported:
```

Reply:
The way it's rendered right now looks nice. Is there a reason to split this?
https://user-images.githubusercontent.com/7101064/146656244-41c87a5c-e10a-44ff-be04-0462f7755402.png

Reply:
I think the rule that the parameter description should be one short sentence, and the follow up should be in the new line.
```
key - Specifies, which property is requested.
The following properties are supported:
....
```
We do that for other ops.

Reply:
Done